src/amo/components/HeroRecommendation/index.js

```diff
@@ -184,6 +195,51 @@ export class HeroRecommendationBase extends React.Component<InternalProps> {
       loading = true;
     }
 
+    const renderHeroTitle = () => {
+      // translators: If uppercase does not work in your locale,
+      // change it to lowercase. This is used as a secondary heading.
+      let titleText = i18n.gettext('SPONSORED');
```

Comment:
This is tricky because of l10n. Somewhere in our code we need to call `i18n.gettext('SPONSORED')` in order to flag this for translators, and we can't really do that from a constants.js file as that won't have access to the current language. I'm not sure we can do much better than this.